sparkmagic/sparkmagic/utils/utils.py

```diff
@@ -53,6 +56,29 @@ def coerce_pandas_df_to_numeric_datetime(df):
             except (ValueError, TypeError):
                 pass
 
+
+def records_to_dataframe(records_text, kind):
```

Comment:
😞 It seems like we were indirectly testing that function by having empty content, invalid content, and etcetera go through the function.

It's OK for now to leave it untested. Please just create an issue specifying that the function has no unit tests.